### ISSUE TITLE
Make the default image larger

### ISSUE THREAD

**Issue by sindresorhus** [beatfreaker/text-meme#1]
A lot of people have high resolution (retina) screens and default size becomes very small when use make it use half the width (to get double the resolution).

Like here: https://github.com/beatfreaker/text-meme-cli/pull/1#discussion_r48091644

**Comment by beatfreaker** [comment]
@sindresorhus so the default size should be 160 right?

**Comment by sindresorhus** [comment]
No, the double of the size you want to use as it needs to be shrunk by the `img` tag to half to the size to be retina. So if you want it to be `160px` it needs to be outputted in `320px`. I think `300px` would be a good size, so it needs to be outputted with `600px`.

**Comment by beatfreaker** [comment]
changes default image size from `320` to `600` [check here](https://github.com/beatfreaker/text-meme/blob/master/index.js#L8)
@sindresorhus can you please review it once?

**Comment by sindresorhus** [comment]
Looks good, but you should also document the default size in this readme and in the CLI readme.
